spring-cloud-gcp-autoconfigure/src/main/java/com/google/cloud/spring/autoconfigure/bigquery/GcpBigQueryAutoConfiguration.java

```diff
@@ -134,10 +134,11 @@ public BigQueryTemplate bigQueryTemplate(
       BigQuery bigQuery,
       BigQueryWriteClient bigQueryWriteClient,
       @Qualifier("bigQueryThreadPoolTaskScheduler")
-          ThreadPoolTaskScheduler bigQueryThreadPoolTaskScheduler) {
+      ThreadPoolTaskScheduler bigQueryThreadPoolTaskScheduler) {
     Map<String, Object> bqInitSettings = new HashMap<>();
     bqInitSettings.put("DATASET_NAME", this.datasetName);
     bqInitSettings.put("JSON_WRITER_BATCH_SIZE", this.jsonWriterBatchSize);
+    bqInitSettings.put("JSON_WRITER_THREAD_POOL_SIZE", this.threadPoolSize);
```

Comment:
Is this still needed?

Reply:
No, it was used with the previous ExecutorService. Removed it